### PR TITLE
Bug 1069593 - making usage graph accessible.

### PR DESCRIPTION
**Pull request by yzen**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[yzen Yura Zenevich (yzen)](https://github.com/yzen) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=415935e4bc15)

**Comment by try-server-hook** [comment]
[yzen Yura Zenevich (yzen)](https://github.com/yzen) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=d3ece749e4d1)

File: apps/costcontrol/js/views/appdetail.js [hunk 1]
```diff
@@ -82,7 +82,11 @@ var AppDetailView = (function() {
                 enabled: true
               }
             },
-            todayLabel: {}
+            todayLabel: {},
+            accessibility: {
+              dataLimitValue: settings.dataLimitValue,
```
Comment (gitmai) [inline apps/costcontrol/js/views/appdetail.js]: IMO, these fields should be allocated under model.limits section, it makes the fields easily accessible.

**Comment by try-server-hook** [comment]
[yzen Yura Zenevich (yzen)](https://github.com/yzen) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=48c85ddec0a9)

**Comment by autolander** [comment]
The pull request could not be applied to the integration branch. Please try again after current integration is complete. You may need to rebase your branch against the target branch.

**Comment by try-server-hook** [comment]
[yzen Yura Zenevich (yzen)](https://github.com/yzen) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=b821c88d7a67)

**Pull request merged**